### PR TITLE
[WEB-7296] add getVouchers method to ecom client and new rewards service client

### PR DESCRIPTION
**Pull request by luke-wyeth**

### CODE REVIEW

File: src/Ecom/Command/VoucherList.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        return http_build_query($this->commandArgs);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getArgsDefinition(): array
+    {
+        return [
+            'user_id'       => ['type' => self::ARG_TYPE_INTEGER, 'required' => true]
```
Comment (gabrieljo) [inline src/Ecom/Command/VoucherList.php:64]: Too much space.

File: src/Ecom/EcomClient.php [hunk 2]
```diff
@@ -43,16 +55,17 @@ public function getBaseUri(): string
...
-            'CancelSubscription'       => '\\Serato\\SwsSdk\\Ecom\\Command\\SubscriptionCancel',
-            'CreateInvoice'            => '\\Serato\\SwsSdk\\Ecom\\Command\\InvoiceCreate',
-            'UpdateCartBillingAddress' => '\\Serato\\SwsSdk\\Ecom\\Command\\UpdateCartBillingAddress',
-            'GetVoucherTypes'          => '\\Serato\\SwsSdk\\Ecom\\Command\\VoucherTypeList',
-            'CreateVoucher'            => '\\Serato\\SwsSdk\\Ecom\\Command\\VoucherCreate',
-            'AssignVoucher'            => '\\Serato\\SwsSdk\\Ecom\\Command\\VoucherAssign',
-            'RedeemVoucher'            => '\\Serato\\SwsSdk\\Ecom\\Command\\VoucherRedeem',
-            'UpdateUserBillingAddress' => '\\Serato\\SwsSdk\\Ecom\\Command\\UserBillingAddressUpdate',
-            'GetInvoicesSummary'       => '\\Serato\\SwsSdk\\Ecom\\Command\\InvoicesSummary'
+            'GetSubscriptions'         => SubscriptionList::class,
```
Comment (gabrieljo) [inline src/Ecom/EcomClient.php:58]: Nice one.

**Review by gabrieljo — CHANGES_REQUESTED** [review]
Nice work @luke-wyeth I think you missed unit test for the command `voucherList`

**Review by gabrieljo — APPROVED** [review]
LGTM. Nice work! @luke-wyeth

**Review by electric — APPROVED** [review]
Looks good - nice work @Mike-Serato

**Pull request merged**